aws/resource_aws_cur_report_definition.go

```diff
@@ -0,0 +1,189 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/costandusagereportservice"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+)
+
+func resourceAwsCurReportDefinition() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCurReportDefinitionCreate,
+		Read:   resourceAwsCurReportDefinitionRead,
+		Delete: resourceAwsCurReportDefinitionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"report_name": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
It seems this attribute should be `Required: true` and have `Computed: true` removed.